public/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <link href="https://fonts.googleapis.com/css2?family=M+PLUS+1p:wght@100&display=swap" rel="stylesheet">
```

Comment:
@hamirmahal  Font changes should be done in another PR. Feel free to create another PR to add fonts 😃

Reply:
I moved the font changes commit to #40, but that still might cause issues because I added those changes to @akshara-sun's [Design-Updates](https://github.com/amilajack/freesound-player/tree/Design-Updates) branch. So, I may need to do this again, this time building the commits off `master`.